gradle.properties

```diff
@@ -19,9 +19,11 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+kotlin.native.ignoreDisabledTargets=true
+kotlin.mpp.enableGranularSourceSetsMetadata=true
 
 GROUP=org.reduxkotlin
-VERSION_NAME=0.5.3
+VERSION_NAME=0.5.4-SNAPSHOT
```

Comment:
Adding SNAPSHOT is not needed.  There is a env var that is set to make snapshot builds.  CI will take care of adding the "-SNAPSHOT" appendage.  Releases are published only on tags.

Reply:
Reverted